docs/source/guides/writer/libs/vmimage.rst.data/supported_images.csv

```diff
@@ -0,0 +1,50 @@
+Provider, Version, Architecture
```

Comment:
This is going to be generated automatically, right? Maybe we could add this to the .gitignore file.